policyengine_uk/variables/gov/hmrc/tax_free_childcare/tax_free_childcare_eligibility.py

```diff
@@ -19,7 +19,7 @@ def formula(benunit, period, parameters):
             benunit.members(
                 "tax_free_childcare_meets_income_requirements", period
             )
-            | benunit.members("is_child", period)
+            | ~benunit.members("is_parent", period)
```

Comment:
This is just as vague as `is_child`. @MaxGhenis I'm not sure I see a great alternative to using `is_child` to identify who should be checked for work statuses (and `is_child` works in all cases I think).

Reply:
Is it? `is_parent` seems more explicitly defined than just being 18+ (the not-necessarily-obvious definition of `~is_child`): https://github.com/PolicyEngine/policyengine-uk/blob/master/policyengine_uk/variables/household/demographic/is_parent.py

Reply:
That is - I'd like to be explicit about whether we're enforcing work requirements on
1. the oldest person or oldest two married people in the benunit, or
2. all people age 18+

Even if the benunit construction in the FRS defines them equivalently these do seem like distinct policies

Reply:
OK we should go with 1. @vahid-ahmadi can you create a variable where the oldest person or the oldest two married people would satisfy the condition and use that?

Reply:
(1) is my understanding of the `is_parent` logic